hs_access_control/tests/test_user_delete.py

```diff
@@ -0,0 +1,44 @@
+
+from django.test import TestCase
+from django.contrib.auth.models import Group
+
+from hs_access_control.models import UserAccess
+
+from hs_core import hydroshare
+from hs_core.testing import MockIRODSTestCaseMixin
+from hs_access_control.tests.utilities import global_reset
+
+
+class T12UserDelete(MockIRODSTestCaseMixin, TestCase):
+    def setUp(self):
+        super(T12UserDelete, self).setUp()
+
+        global_reset()
+        self.group, _ = Group.objects.get_or_create(name='Hydroshare Author')
+        self.admin = hydroshare.create_account(
+            'admin@gmail.com',
+            username='admin',
+            first_name='administrator',
+            last_name='couch',
+            superuser=True,
+            groups=[]
+        )
+
+        self.cat = hydroshare.create_account(
+            'cat@gmail.com',
+            username='cat',
+            first_name='not a dog',
+            last_name='last_name_cat',
+            superuser=False,
+            groups=[]
+        )
+
+    def test_00_cascade(self):
```

Comment:
test_delete.py handles part of that. Since the default action for the others is cascade, the only question left open was whether a one-to-one relationship cascades. When one deletes a UserAccess, its membership cascade deletes the others. If you want this more explicitly tested, it is possible. 

Cascade can do some bad things if the user owns resources.